css-basic/css-basics_pink-box/css/styles.css

```diff
@@ -9,9 +9,25 @@ body {
 }
 
 /* put a selector here to select div elements (see index.html) */
+
+.text {
+  font-size: large;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  text-emphasis-color: tomato;
+  
+}
+
+
 SELECTOR HERE {
```

Comment:
http://htmlpreview.github.io/?https://github.com/Nicki-lombardo/Challenges/blob/pink-box/css-basic/css-basics_pink-box/index.html  Styles are not applied to the block, there is no selector. You need to specify a selector of type or in your case class .text

Reply:
I don't understand. I could rephrase his idea. thank you!

Reply:
The block is not pink, the styles you wrote don't apply to the block, you need to replace ‘SELECTOR HERE’ with the div or class selector you added.